app/src/main/java/com/vultisig/wallet/ui/models/QrAddressViewModel.kt

```diff
@@ -69,7 +69,9 @@ internal class QrAddressViewModel @Inject constructor(
         val qrBitmap = withContext(Dispatchers.IO) {
             makeQrCodeBitmapShareFormat(bitmap, color, logo, title, null)
         }
-        shareQrBitmap.value?.recycle()
+        if (shareQrBitmap.value != null && shareQrBitmap.value?.isRecycled != true) {
+            shareQrBitmap.value?.recycle()
```

Comment:
<img width="669" alt="image" src="https://github.com/user-attachments/assets/277d8d3c-6712-4dbb-84ba-581028b855ac">

Based on the document , it seems we should not call it , and instead just let GC to do it's job

Reply:
recycle is just a request , doesn't mean it will be recycled immediately